uber/decorators.py

```diff
@@ -452,13 +454,20 @@ def food_price(self):
     """
 
 
-class class_property(object):
-    """Read-only property for classes rather than instances."""
-    def __init__(self, func):
-        self.func = func
-
-    def __get__(self, obj, owner):
-        return self.func(owner)
+class cached_classproperty(property):
+    """
+    Like @cached_property except it works on classes instead of instances.
+    """
+    def __init__(self, fget, *arg, **kw):
+        super(cached_classproperty, self).__init__(fget, *arg, **kw)
```

Comment:
Is this line meant to call the init function of a parent `cached_classproperty`?